pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -107,6 +110,16 @@ func status() registry.State {
 			glog.Warningf("Warning ! minimum required version for podman is %s. your version is %q. minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html ", minReqPodmanVer.String(), v.String())
```

Comment:
If they install via brew this will now lead to an error.

```
$ brew cask info podman
podman: 2.0.2
```

Should I update the doc to github 1.9 release for these checks too?